nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/logback.xml

```diff
@@ -95,6 +107,9 @@
             <maxFileSize>10MB</maxFileSize>
             <maxHistory>10</maxHistory>
             <totalSizeCap>100MB</totalSizeCap>
+            <totalSizeCap>5GB</totalSizeCap>
```

Comment:
This `5GB` setting duplicates the existing value of `100MB`. In this particular case, the setting for `nifi-deprecation.log` is intentionally smaller to avoid excessive repetition.